### PR TITLE
chore: add changelog doc and testnet Fermat height

### PR DESCRIPTION
**Pull request by owen-reorg**
### Description

Add changelog doc and testnet Fermat height.

### Rationale

NA

### Example

NA

### Changes

NA

### CODE REVIEW

**Review by welkin22 — DISMISSED** [review]
LGTM

**Review by redhdx — DISMISSED** [review]

**Review by 2129zxl — APPROVED** [review]
LGTM for the content related to P2P Engine sync

**Review by krish-nr — DISMISSED** [review]
LGTM for the content related to P2P Engine sync

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## v0.2.0
+
+This is a hardfork release for the opBNB Testnet called Fermat.
+It will be activated at block height 13063000, expected to occur on November 14, 2023, at 6 AM UTC.
+
+### User Facing Changes
+
+- The L1 gas price of all L2 transactions will be fixed(3 Gwei by default and 5 Gwei for Testnet).
+- Introduce a new type of RPC kind called `bsc_fullnode`. To enable it, include the parameter `--l1.rpckind=bsc_fullnode` if the layer 1 endpoint supports the `eth_getTransactionReceiptsByBlockNumber` API. This will significantly enhance the performance of retrieving L1 receipts (#63).
+- The rollup configuration for opBNB Mainnet and Testnet has been added to the code. You can now use the `--network=opBNBTestnet` or `--network=opBNBMainnet` flag instead of `--rollup.config=./rollup.json` to specify the rollup configuration for the op-node. (#65)
```
Comment (bendanzhentan) [inline CHANGELOG.md:12]: If a node still runs with `--rollup.config=./rollup.json` and the `./rollup.json` doesn't update to date, will the node fork against the main network?

Comment (owen-reorg) [reply CHANGELOG.md:12]: It will retrieve the block chain id in `rollup.json` and overwrite the config it the chain id matches Mainnet or Testnet

**Review by bendanzhentan — APPROVED** [review]

**Review by welkin22 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Full Changelog
+
+https://github.com/bnb-chain/opbnb/compare/v0.1.2...v0.2.0
+
+## v0.1.2
+
+This is the initial release for opBNB Testnet.
+
+The repo base is [optimism v1.0.8](https://github.com/ethereum-optimism/optimism/commits/v1.0.8).
```
Comment (nolanxyg) [inline CHANGELOG.md:41]:
```suggestion
The repo base is [optimism v1.1.0](https://github.com/ethereum-optimism/optimism/releases/tag/op-node%2Fv1.1.0).
```

**Pull request merged**